src/containers/footer-links/footer-links.styles.js

```diff
@@ -0,0 +1,40 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export const useStyles = makeStyles(() => ({
+  cardBody: {
+    flex: '1 1 auto',
+    minHeight: '1px',
```

Comment:
What will you see in 1px? make it more or remove it

Reply:
Removed